examples/big-bang/template/bigbang/vendor/bigbang/base/flux/kustomization.yaml

```diff
@@ -33,7 +33,7 @@ patches:
               cluster-autoscaler.kubernetes.io/safe-to-evict: "true"
           spec:
             imagePullSecrets:
-            - name: private-registry
+            - name: zarf-registry
```

Comment:
This change gets blown away with our current Make recopies. `make package-example-big-bang` calls the `vendor-big-bang-base` recipe which pulls down the remote tagged version of BB which uses `private-registry` as the pull secret. 

My questions are:
  1. Should we be checking in vendored code to this repo?
  2. If we're checking in vendored code so that we can modified values like this do we need that make recipie that re-pulls with every build?

Reply:
we can't modify anything in the `.../vendor` folder since it will get blown away by the build process.

Reply:
Given that Flux is not able to pull private bases we need to vendor the code, unless some other solution is come up with

Reply:
I may need some help understanding this one.  Big bang has a value that will let you define what the image pull cred is last time I checked, am I missing something?  I'm also super confused by how complex our flux manifest section is in general for the BB examples.

Reply:
Okay I have a solution for this and will be working tonight / tomorrow to wrap it up for both bb example packages.  Will update when it's done.

Reply:
this is resolved now with the addition of kustomization sources + serviceaccount updates to use the zarf-registry on zarf+helm deploy